### PR TITLE
High contrast light + dark themes

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ yarn change`

#### Description of changes
Added 2 new themes to Azure-themes, HighContrastLight and HighContrastDark. Now have 4 themes. 

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by Jacqueline-ms** [comment]
## General update: 
updated button height to 24px from 36px

## High Contrast Light theme:
Note: choice group and checkboxes in portal high contrast light theme showed regular light theme coloring (Azure blue) instead of the high contrast light purple accent color. So I did not deviate 
image
image
image
image
image
image
image
image
image

## High Contrast Dark theme:
image
image
image
image
image
image
image
image
image

File: packages/azure-themes/src/azure/AzureThemeDark.ts [hunk 1]
```diff
@@ -80,6 +106,13 @@ const darkExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
   statusWarningBackground: DarkSemanticColors.statusBar.warning,
   statusWarningText: DarkSemanticColors.text.body,
   statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: DarkSemanticColors.primaryButton.disabled.border,
+
+  // temporary work around for high contrast themes
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/AzureThemeDark.ts:111]: Are you planning to fix this temporary workaround? If so, please create a workitem and mention the workitem number here.

File: packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  statusSuccessBackground: HighContrastLightSemanticColors.statusBar.okay,
+  statusSuccessText: HighContrastLightSemanticColors.text.body,
+  statusSuccessIcon: CommonSemanticColors.icons.okay,
+  statusWarningBackground: HighContrastLightSemanticColors.statusBar.warning,
+  statusWarningText: HighContrastLightSemanticColors.text.body,
+  statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: HighContrastLightSemanticColors.text.disabled,
+  textFieldBackgroundDisabled: HighContrastLightSemanticColors.disabledButton.background,
+
+  // temporary work around for high contrast themes
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts:112]: Same as above comment. Mention a workitem number to fix this workaround, if applicable.

File: packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  },
+  palette: {
+    themePrimary: HighContrastLightSemanticColors.controlOutlines.accent,
+    neutralPrimary: HighContrastLightSemanticColors.text.body,
+    neutralDark: HighContrastLightSemanticColors.text.body,
+    neutralLight: HighContrastLightSemanticColors.shimmer.secondary, // shimmer elements
+    neutralLighter: HighContrastLightSemanticColors.shimmer.primary, // shimmer elements
+    neutralLighterAlt: HighContrastLightSemanticColors.item.hover, // nav highlight
+    neutralQuaternaryAlt: HighContrastLightSemanticColors.item.select, // expand button on list controls
+    neutralSecondary: HighContrastLightSemanticColors.text.label, // persona
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts:134]: what do you mean by 'persona' here? elaborate more in the comment so others can understand better?

File: packages/azure-themes/src/azure/AzureThemeLight.ts [hunk 4]
```diff
@@ -80,7 +106,15 @@ const lightExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
   statusWarningBackground: LightSemanticColors.statusBar.warning,
   statusWarningText: LightSemanticColors.text.body,
   statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: LightSemanticColors.disabledButton.background,
+
+  // temporary work around for high contrast themes
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/AzureThemeLight.ts:111]: add workitem if applicable.

File: packages/azure-themes/src/azure/styles/DropDown.styles.ts [hunk 5]
```diff
@@ -44,7 +43,7 @@ export const DropdownStyles = (props: IDropdownStyleProps): Partial<IDropdownSty
 
         selectors: {
           ':focus::after, :focus': {
-            borderColor: semanticColors.primaryButtonBackground,
+            //borderColor: semanticColors.inputBorderHovered,
```
Comment (hyoshis) [inline packages/azure-themes/src/azure/styles/DropDown.styles.ts:46]: remove commented code.

**Pull request merged**